app/javascript/widgets/landing_page/components/layout/PageHead.jsx

```diff
@@ -27,17 +28,28 @@ const PageHead = () => {
   return (
     <PageHeader>
       <Stack className="tw-ml-auto" gap="4" alignment="center">
-        {selectedDomain === "CC3TEST" && (
-          <a
-            href={buildPasswordLoginLink(selectedRegion, selectedDomain, prodMode)}
-            className="tw-text-theme-disabled hover:tw-underline"
-          >
-            Log in with password
-          </a>
-        )}
-        <Button variant="primary" size="small" icon="manageAccounts" title="Log in" onClick={handleLoginButtonClick}>
-          Log in
-        </Button>
+        <>
+          {selectedDomain === "CC3TEST" && (
+            <a
+              href={buildPasswordLoginLink(selectedRegion, selectedDomain, prodMode)}
+              className="tw-text-theme-disabled hover:tw-underline"
+            >
+              Log in with password
+            </a>
+          )}
+          {hideDomainSwitcher === "false" ||
+            (hideDomainSwitcher === false && (
```

Comment:
`hideDomainSwitcher` is a boolean value from the way it's set in `App.jsx` so it should be enough to just check whether it's falsy here:
```
{!hideDomainSwitcher && (
   <Button
    .....
```

Reply:
👍